google/cloud/logging/_gax.py

```diff
@@ -18,18 +18,17 @@
 
 from google.gax import CallOptions
 from google.gax import INITIAL_PAGE
-from google.gax.errors import GaxError
 from google.logging.type.log_severity_pb2 import LogSeverity
 from google.logging.v2.logging_config_pb2 import LogSink
 from google.logging.v2.logging_metrics_pb2 import LogMetric
 from google.logging.v2.log_entry_pb2 import LogEntry
 from google.protobuf.json_format import Parse
 from grpc import StatusCode
+from grpc._channel import _Rendezvous
```

Comment:
Argggh!  Are we really going to commit to scattering use of that private name everywhere?  Could we mitigate by importing it into `google.cloud.exceptions`, and then aliasing it with a name like `GrpcRendezvous`?